src/components/GridLayout.js

```diff
@@ -54,6 +55,28 @@ class GridLayout extends Component {
   componentWillMount() {
```

Comment:
componentWillMount is legacy, misschien wordt de code eenvoudiger als we dit gewoon in de constructor stoppen?